WalletWasabi.Gui/Tabs/WalletManager/RecoverWalletView.xaml

```diff
@@ -29,6 +29,18 @@
               </DataTemplate>
             </ItemsControl.ItemTemplate>
           </ItemsControl>
+          <StackPanel Spacing="16">
+            <CheckBox IsChecked="{Binding ShowAdvancedOptions,Mode=TwoWay}">
+              <TextBlock Text="Show advanced options"/>
+            </CheckBox>
+            <StackPanel IsVisible="{Binding ShowAdvancedOptions}">
+              <controls:ExtendedTextBox Text="{Binding AccountKeyPath}" Watermark="Account Key Path" UseFloatingWatermark="True"/>
+              <TextBlock Text="Note that, Wasabi can only monitor native SegWit (bech32) addresses."/>
+            </StackPanel>
+            <StackPanel IsVisible="{Binding ShowAdvancedOptions}">
+              <controls:ExtendedTextBox Text="{Binding MinGapLimit}" Watermark="Minimum Gap Limit" UseFloatingWatermark="True"/>
```

Comment:
It could worth to use a `NumericUpDown` control for the MinGapLimit because it makes easier to limit min and max and also make sure that a numerical value is entered.

Reply:
I tried that. But then we don't have that nice floating watermark. And I noticed that if the property `MinGapLimit` is of type `int` then the ExtendedTextBox automatically parses the entered text as a number and also shows default error messages. So we only need to check upper and lower bound as is done in `RecoverWalletViewModel`.
So I didn't change it at the moment, hope that's ok.

Reply:
okay.